infrastructures/infrastructure-azure/src/main/java/org/ow2/proactive/resourcemanager/nodesource/billing/AzureBillingResourceUsage.java

```diff
@@ -269,15 +364,23 @@ public LocalDateTime getResourceUsageReportedEndDateTime() {
         return this.resourceUsageReportedEndDateTime;
     }
 
-    public double getVmGlobalCost() {
-        return this.vmGlobalCost;
+    public double getGlobalCost() {
+        return this.globalCost;
     }
 
-    public void setVmGlobalCost(double vmGlobalCost) {
-        this.vmGlobalCost = vmGlobalCost;
+    public double getBudgetPercentage() {
+        return this.budgetPercentage;
+    }
+
+    public void setGlobalCost(double globalCost) {
+        this.globalCost = globalCost;
     }
 
     public void setCurrency(String currency) {
         this.currency = currency;
     }
+
+    public void setBudgetPercentage(double budgetPercentage) {
+        this.budgetPercentage = budgetPercentage;
+    }
```

Comment:
You can use lombok instead of writing getters and setters.

Reply:
Ok .. see changes in the next commit.